server/utils/index.js

```diff
@@ -13,6 +13,7 @@ import commentValidator from './comments/comments.util';
 import newArticleMail from './notifications/article-mail.util';
 import newFollowerMail from './notifications/follows-mail.util';
 import newCommentMail from './notifications/comment-mail.util';
+import verifyHighlightedText from './articles/highlightvalidate.util';
```

Comment:
You could have destructured the verifyHighlightedText so you get all you need from it inside the utils index file, how about that?

Reply:
Noted @davidokonji.